packages/css-engine/src/core/to-value.ts

```diff
@@ -68,15 +68,15 @@ export const toValue = (
   }
 
   if (value.type === "image") {
-    if (value.hidden) {
+    if (value.hidden || value.value.type !== "url") {
       // We assume that property is background-image and use this to hide background layers
       // In the future we might want to have a more generic way to hide values
       // i.e. have knowledge about property-name, as none is property specific
       return "none";
     }
 
     // @todo image-set
-    return `url(${value.value.value.path}) /* id=${value.value.value.id} */`;
+    return `url(${value.value.url})`;
```

Comment:
if we really need that id for debugging we could keep it on the value

Reply:
Idk, @istarkov do we need it?

Reply:
no